src/alert/alert.ts

```diff
@@ -63,6 +65,10 @@ export class NgbDismissibleAlert implements OnInit {
    * An event emitted when the close button is clicked.
    */
   @Output('close') closeEvent = new EventEmitter();
+  /**
+   * An event emitted when the close button is clicked.
```

Comment:
Change the comment to fit the input description, like "Amount of time in milliseconds before the alert is self closing" Thx